reference/7.1/Microsoft.PowerShell.Utility/Sort-Object.md

```diff
@@ -82,7 +83,8 @@ The `Get-ChildItem` cmdlet gets the files and subdirectories from the directory
 This command displays the files in the current directory by length in ascending order.
 
 ```
-PS> Get-ChildItem -Path C:\Test -File | Sort-Object -Property Length
+PS> Get-ChildItem -Path C:\Test -File | Sort-Object 
+ty Length
```

Comment:
Are you sure?  It looks like a typing accident.

Reply:
Thanks. That was an accident.

Reply:
Fixed